src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> readFromFile(String filePath) {
+        List<String> dataFromFile = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                dataFromFile.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("File doesn't exist", e);
```

Comment:
The exception message is not informative enough. It should include the path to the file that caused the exception. For example: "Can't read from file: " + filePath